modifytags/modifytags.go

```diff
@@ -286,7 +285,17 @@ func (mod *Modification) addTags(fieldName string, tags *structtag.Tags) (*struc
 	case Keep:
 		name = fieldName
 	default:
-		unknown = true
+		// Use snakecase as the default.
```

Comment:
Use `fallthrough` to avoid duplicating logic here.

Also, any observable behavior change like this should have a corresponding change in documentation.